modules/administration/pages/auth-methods-sso.adoc

```diff
@@ -0,0 +1,97 @@
+[[auth-methods]]
+= Authentication With Single Sign-On (SSO)
+
+{productname} supports single sign-on (SSO) by implementing the Security Assertion Markup Language (SAML){nbsp}2 protocol.
```

Comment:
I discovered yesterday that SSO only works for the WebUI, but not for console commands.
We should add a big warning about it.

Reply:
I believe there's a separate issue about that.